--- koji-1.11.0-orig/cli/koji.conf	2016-12-08 22:22:14.000000000 -0600
+++ koji-1.11.0/cli/koji.conf	2016-12-09 08:29:10.066764435 -0600
@@ -3,26 +3,17 @@
 ;configuration for koji cli tool
 
 ;url of XMLRPC server
-;server = http://hub.example.com/kojihub
+server = https://koji.fedoraproject.org/kojihub
 
 ;url of web interface
-;weburl = http://www.example.com/koji
+weburl = https://koji.fedoraproject.org/koji
 
 ;url of package download site
-;pkgurl = http://www.example.com/packages
+topurl = https://kojipkgs.fedoraproject.org/
 
 ;path to the koji top directory
 ;topdir = /mnt/koji
 
-;configuration for Kerberos authentication
+authtype = kerberos
+krb_rdns = false

-;the service name of the principal being used by the hub
-;krbservice = host
-
-;configuration for SSL authentication
-
-;client certificate
-;cert = ~/.koji/client.crt
-
-;certificate of the CA that issued the HTTP server certificate
-;serverca = ~/.koji/serverca.crt
